src/components/common/needsShutdown.jsx

```diff
@@ -159,7 +159,10 @@ export const VmNeedsShutdown = ({ vm }) => {
             position="bottom"
             hasAutoWidth
             bodyContent={body}>
-            <Label className="resource-state-text" href="javascript:null" color="blue" id={`vm-${vm.name}-needs-shutdown`} icon={<PendingIcon />}>{_("Changes pending")}</Label>
+            <Label className="resource-state-text" color="blue" id={`vm-${vm.name}-needs-shutdown`}
+                   icon={<PendingIcon />} onClick={() => null}>
```

Comment:
Wondering if dropping `onClick` is enough?

Reply:
I *added* it. It needs to have either a `href=` or an `onClick` handler to become focusable, and I think that was the original intent for that hack.